docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -144,34 +144,45 @@ all labeled and additional information will be included below.
 
 
 .. list-table:: Outputs from Pregnancy Decision Tree
-  :widths: 3 15 15
+  :widths: 3 15 15 15
   :header-rows: 1
 
   * - Input
     - Data Source 
-    - Notes
+    - Overall stratification
+    - Use case 
   * - ANC attendance
     - Decision tree point
-    - 
+    - Age, scenario 
+    - Decision tree value & export for V&V
   * - Ultrasound status (AI assisted, standard, none)
     - Decision tree point
-    - 
+    - Scenario
+    - Decision tree value & export for V&V
   * - Gestational age at birth
     - :ref:`Pregnancy model <other_models_pregnancy_closed_cohort_mncnh>`
-    - 
+    - Decision tree value & export for V&V 
+    - Scenario
   * - Gestational age stated
-    - Decision tree value
-    - We should track both the real GA at birth and the believed GA 
-  * - Birthweight
-    - :ref:`Pregnancy model <other_models_pregnancy_closed_cohort_mncnh>`
+    - Decision tree value 
     - 
-  * - Low birthweight status
+    - Decision tree value and & export for V&V 
+  * - Birthweight
     - :ref:`Pregnancy model <other_models_pregnancy_closed_cohort_mncnh>`
     - 
+    - Decision tree value & & export for V&V  
   * - Pregnancy term (full term or partial term)
     - :ref:`Pregnancy model <other_models_pregnancy_closed_cohort_mncnh>`
     - 
+    - Decision tree value
+  * - Age of pregnant simulant
```

Comment:
When we say "Age of pregnant simulant," do we want their exact age, their GBD age group, or both?